app/services/identity_linker.rb

```diff
@@ -9,6 +9,7 @@ def initialize(user, provider)
   def link_identity(**extra_attrs)
     attributes = merged_attributes(extra_attrs)
     identity.update!(attributes)
+    AgencyIdentityLinker.new(identity).link_identity if FeatureManagement.enable_agency_based_uuids?
```

Comment:
What does the call to `AgencyIdentityLinker` here do if you're still returning the old identity at the end of the method? Did you mean to do this instead?
```ruby
if FeatureManagement.enable_agency_based_uuids?
  AgencyIdentityLinker.new(identity).link_identity
else
  identity
end
```

Reply:
No. The AgencyIdentityLinker returns an AgencyIdentity. This method is expecting an identity to be returned.

Reply:
If we're not using the return value of the AgencyIdentityLinker, what is the purpose of calling it here?

Reply:
When we flip it live this linker needs to create an agency_identity row to establish a definitive uuid for a user even if the agency associated with the user is not enabled to return the new uuids.

Reply:
I see. Would it make sense to split the creation portion of `AgencyIdentityLinker#link_identity` into a separate method and then call that here since all we need here is to create the agency_identity and not have one returned to us?

Reply:
Would renaming link_identity to find_or_create_agency_identity make it more clear?  Then I can just rename the private version.

Reply:
I think it's fine for now. I'll have to think about this some more.